### PR TITLE
Design of Salary input and autofocus

### PR DESCRIPTION
**Pull request by ramyamahi**

### CODE REVIEW

**Comment by ramyamahi** [comment]
@jarjan New design completed fully with responsive. Please merge it
new-design

File: src/App.js [hunk 1]
```diff
@@ -44,53 +44,65 @@ class App extends Component {
...
-            <dt>Заработная плата за год:</dt>
-            <dd>{round(grossSalary * 12)}</dd>
-
-            <dt>Оклад за год:</dt>
-            <dd>{round(netSalary * 12)}</dd>
-          </dl>
-          <hr />
+          <section className="App-input-section">
+            <p>Введите свою заработную плату в месяц в теңге:</p>
+            <section className="App-form">
```
Comment (jarjan) [inline src/App.js]: I suggest to change `section` tag with `div` for now.

File: src/App.js [hunk 2]
```diff
@@ -44,53 +44,65 @@ class App extends Component {
...
+              <input
+                className="App-checkbox"
+                type="checkbox"
+                id="net"
+                checked={net}
+                onChange={this.handleCheckboxChange}
+              />
+            </section>
+          </section>          
+          <table className="App-list">
```
Comment (jarjan) [inline src/App.js]: And maybe change `className` to `App-table`.

File: src/App.js [hunk 3]
```diff
@@ -44,53 +44,65 @@ class App extends Component {
...
+            <tbody>  
+              <tr className="App-row">
+                <td>
+                  <p>Оклад</p>
+                  <p>{round(netSalary)}</p>
```
Comment (jarjan) [inline src/App.js]: I think this line could be separate table row.

File: src/App.js [hunk 4]
```diff
@@ -44,53 +44,65 @@ class App extends Component {
...
+                </td>
+                <td>
+                  <p className="App-list-right"><abbr title="Обязательный пенсионный взнос">ОПВ</abbr></p>
+                  <p className="App-list1 App-list-right">{round(pension)}</p>
```
Comment (jarjan) [inline src/App.js]: Here also separate row, and aligning can be done by adding className to `td` tag.

File: src/App.js (hunk 1 above)
Comment (ramyamahi) [reply src/App.js]: @jarjan  Done

File: src/App.js (hunk 2 above)
Comment (ramyamahi) [reply src/App.js]: @jarjan Done

File: src/App.js (hunk 3 above)
Comment (ramyamahi) [reply src/App.js]: @jarjan Do you want Salary as seperate row? We aligned items as horizontally. Each table row has two items. (For example: Salary and Pension)

Comment (jarjan) [reply src/App.js]: @ramyamahi yes, separate each items to rows. This way we can remove `p` tag and put content insider `td` directly.

File: src/App.js (hunk 4 above)
Comment (ramyamahi) [reply src/App.js]: @jarjan Done

File: src/App.js (hunk 3 above)
Comment (ramyamahi) [reply src/App.js]: Done

**Pull request merged**